### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by zeitgenossek2** [matthewmiglio/py-clash-bot#189]
the bot does the same thing over and over again. closing memu, opening memu, closing memu, opening memu.... endless

**Comment by marmig0404** [comment]
is this the same issue as described in #167 ?

**Comment by zeitgenossek2** [comment]
no. its just a loop. closing memu and opening memu over and over.

**Comment by marmig0404** [comment]
can you share which version you're running and what outputs are written to the readout?

**Comment by marmig0404** [comment]
try the [latest stable release v1.1.1](https://github.com/matthewmiglio/py-clash-bot/releases/tag/v1.1.1).
all the version 2.0 stuff is unstable and is in testing right now.

**Comment by zeitgenossek2** [comment]
yea tried it. it stucks at "starting memu launcher"
it opens up multi memu and then nothing happens.

**Comment by zeitgenossek2** [comment]
[https://ibb.co/B6h95DR](url)

**Comment by marmig0404** [comment]
it should be trying to create a MEmu instance at this step, can you confirm you are running as administrator?

**Comment by zeitgenossek2** [comment]
i reinstalled both and now there is an instance. so i manually start the instance and log in the my clash account?

**Comment by zeitgenossek2** [comment]
alright i give up. it does nothing.

**Comment by marmig0404** [comment]
> so i manually start the instance and log in the my clash account?

yes after installing clash you log in. it should be notifying you once it boots up the instance to download and log in if it sees that you don't have CR installed.
i don't have much time to debug this right now but thanks for trying. when i find out a reason behind it i will mention it here.

**Comment by zeitgenossek2** [comment]
it stucks at "starting memu launcher" 
ok thanks for the help.

**Comment by matthewmiglio** [comment]
This issue has been solved. 

However, admittedly, the bot is very buggy since the February update and I still need to handle some of the new clash GUI changes.

The bot can still handle its basic operations (like card upgrading and fight starting). 

TLDR; this issue is fixed, but the bot will still be very buggy for now

**Comment by marmig0404** [comment]
This issue should be fixed in [v1.1.2](https://github.com/matthewmiglio/py-clash-bot/releases/tag/v1.1.2)
